resources/pt-br.yml

```diff
@@ -29,9 +29,9 @@ description: >
 # Amarelo = &e           Branco = &f
 
 
-default_towny_prefix: '&6[Towny] &b'
-default_town_prefix: '&6[%s] &b'
-default_nation_prefix: '&6[%s] &6'
+default_towny_prefix: '&9Towny> &7'
+default_town_prefix: '&6Cidade> &b[%s] &7'
+default_nation_prefix: '&2Nação> &b[%s] &7'
```

Comment:
These lines should be brought back to the normal format. This looks like something your own server does to customize Towny but I doubt every Brazilian server will want it this way.

Reply:
Oops, sorry, this fork is to my server, and I forgot to adjust this.